utilities/remove_expired.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+""" Script to remove prefixes that has expired in NIPAP
+
+    This script removes prefixes in NIPAP with status quarantine
+    and that has expired.
+
+    Usage:
+
+    When the script is executed, it deletes all prefixes with status 
+    "quarantine" and where the expires date has passed.
+
+    Prefixes within prefixes are recursively deleted only if the
+    type is Assigned.
+"""
+
+import ConfigParser
+import os
+import datetime
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
+from pynipap import Prefix
+import pynipap
+import socket
+import xmlrpclib
+
+
+def _remove_expired_prefixes():
+    query = {
+        'operator': 'and',
+        'val1': {
+            'operator': 'equals',
+            'val1': 'status',
+            'val2': 'quarantine'
+        },
+        'val2': {
+            'operator': 'less_or_equal',
+            'val1': 'expires',
+            'val2': 'Now()'
+        }
+    }
+
+    try:
+        res = Prefix.search(query)
+    
+    except socket.error:
+        print >> sys.stderr, "Connection refused, please check hostname and port"
+        sys.exit(1)
+    except xmlrpclib.ProtocolError:
```

Comment:
Hm, won't an authentication error raise some NipapAuthError?

Reply:
Yes, I changed to NipapAutError and added also a general exception.